website/docs/r/rde_type.html.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_rde_type"
+sidebar_current: "docs-vcd-resource-rde-type"
+description: |-
+   Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+---
+
+# vcd\_rde\_type
+
+Provides the capability of creating, updating, and deleting Runtime Defined Entity types in VMware Cloud Director.
+Requires system administrator privileges.
+
+Supported in provider *v3.9+*
+
+## Example Usage with a schema file
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  nss     = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema        = file("${path.module}/schemas/my-type-schema.json")
+}
+```
+
+## Example Usage with a URL that contains a schema file
+
+```hcl
+data "vcd_rde_interface" "my_interface" {
+  vendor  = "bigcorp"
+  ns      = "tech1"
+  version = "1.2.3"
+}
+
+resource "vcd_rde_type" "my_rde_type" {
+  vendor        = "vmware"
+  nss           = "vcd"
+  version       = "4.5.6"
+  name          = "My VMware RDE Type"
+  interface_ids = [data.vcd_rde_interface.my_interface.id]
+  schema_url    = "https://just.an-example.com/schemas/my-type-schema.json"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
```

Comment:
Looks like we need a similar message here about the 3-tuple as you had here:
https://github.com/vmware/terraform-provider-vcd/blob/main/website/docs/r/rde_interface.html.markdown#argument-reference

Reply:
Added.